lib/src/components/NxDropdown/__tests__/NxDropdown.test.tsx

```diff
@@ -90,8 +93,10 @@ describe('NxDropdown', () => {
 
   it('uses VDOM as label if supplied', function() {
     const label = <NxFontAwesomeIcon icon={faTrash} />,
-        component = getShallowComponent({ label }),
-        button = component.find(NxButton);
+        component = getMountedComponent({ label });
+
+    const dropdown = component.find(AbstractDropdown);
```

Comment:
Is there a need for this change? `.find` searches the whole subtree

Reply:
> Is there a need for this change? `.find` searches the whole subtree

I forgot to remove this, fixed here: 49aa3582e3f83104b1fe96ee9d123b417d169b53